source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,401 @@
-
-This must result in:
-
-* 16 Controlled Total_Item_Requests and 16 OA_Gold Total_Item_Requests in the TR_B3 Standard View.
-* 8 Controlled Unique_Item_Requests and 8 OA_Gold Unique_Item_Requests in the TR_B3 Standard View.
-* 4 Controlled Unique_Title_Requests and 4 OA_Gold Unique_Title_Requests in the TR_B3 Standard View.
-
-**Option 2**: Content provider does not offer OA_Gold items.
+Within the Title Report, breakdowns by Access_Type are essential. There are therefore a series of audit tests designed to determine report providers’ compliance with requirements for reporting Access_Type. These tests only apply where Journals with more than one Access_Type are available on a platform.
 
-The auditor must carry out a total of 32 tests, and each test will consist of 2 requests. There are 2 types of tests that must be carried out:
+**Option 1**: Report provider offers journal articles under two Access_Types.
 
-* “Inside” tests (Two requests are made to the same book item and the second request is made within 30 seconds of the first).
-* “Outside” tests (Two requests are made to the same book item, and the second request is made over 30 seconds after the first).
+The auditor MUST request 50 journal articles with each Access_Type.
 
-The auditor must carry out 16 **inside** tests.
+This MUST result in 50 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with each Access_Type.
 
-* Where the site allows, each book must have 2 item tests. This will report 2 Total_Item_Requests and 2 Unique_Item_Requests and 1 Unique_Title_Requests.
+The Access_Type combinations might be: Controlled plus Open, Controlled plus Free_To_Read, or Open plus Free_To_Read
 
-This must result in:
+**Option 2**: Report provider offers Controlled, Open and Free_To_Read journal articles.
 
-* 16 Controlled Total_Item_Requests in the TR_B3 Standard View.
-* 16 Controlled Unique_Item_Requests in the TR_B3 Standard View.
-* 8 Controlled Unique_Title_Requests in the TR_B3 Standard View.
+The auditor MUST request
 
-The auditor must carry out 16 **outside** tests.
+* 40 journal articles with Access_Type Controlled.
+* 40 journal articles with Access_Type Open.
+* 20 journal articles with Access_Type Free_To_Read.
 
-* Where the site allows, each book must have 2 item tests. This will report 4 Total_Item_Requests, 2 Unique_Item_Requests, and 1 Unique_Title_Requests.
+This MUST result in 40 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with Access_Type Controlled; the same again for Access_Type Open; and 20 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with Access_Type Free_To_Read.
 
-This must result in:
+**Option 3**: Report provider offers journal articles under just one Access_Type.
```

Comment:
Same suggestion here as for the books, order should be one/two/three Access_Types.